capture/src/disk.cpp

```diff
@@ -53,10 +59,30 @@ void write_to_disk(
 
         if (disk_write_enabled)
         {
+            // Check free disk space (but not every single frame)
+            if (frame_count % 100 == 0)
+            {
+                int rtn = statvfs("/", &disk_stats);
```

Comment:
Should probably make this query the path where the file is, rather than arbitrarily hard-coding the root filesystem.

Reply:
Good call. Replaced `"/"` with `filename` so it will always match whatever filesystem the SER file is being written to.